app/javascript/mastodon/actions/alerts.js

```diff
@@ -23,23 +25,29 @@ export function clearAlert() {
   };
 };
 
-export function showAlert(title = messages.unexpectedTitle, message = messages.unexpectedMessage) {
+export function showAlert(title = messages.unexpectedTitle, message = messages.unexpectedMessage, message_values = undefined) {
```

Comment:
There is really no need for a default value if the default value is `undefined`, JavaScript doesn't care how many arguments you pass into a function, if you pass fewer the other ones are undefined.